### PR TITLE
Add four more stretch goals

### LINKED ISSUE #1: [Week 3] Array Part 1 Only has 1 Stretch Goal
consider asking for more to get Exceeds

### PR DESCRIPTION
**Pull request by eschwartz**
Resolves #1 

These stretch goals are pushing students to look up built-in array methods that we haven't explicitly taught them.

Will update rubric, if this PR is approved

### CODE REVIEW

File: assignment/scripts/array-practice.js [hunk 1]
```diff
@@ -75,3 +75,14 @@ console.log('The animals are now', animalArray);
 
 // 4.d TODO: Remove the food at the beginning of your array & 
 //     log both the food removed and the updated array
+
+// 4.e (STRETCH) TODO: Replace the second food in your array
+//      with another one of your favorite foods.
+//      Then log the updated array.
```
Comment (eschwartz) [inline assignment/scripts/array-practice.js:81]:
```js
favFoods.splice(1, 1, "bananas");
console.log(favFoods);
```

File: assignment/scripts/array-practice.js [hunk 2]
```diff
@@ -75,3 +75,14 @@ console.log('The animals are now', animalArray);
...
+
+// 4.f (STRETCH) TODO: Sort your favoriteFoods array
+//     in reverse alphabetical order. Log the array.
```
Comment (eschwartz) [inline assignment/scripts/array-practice.js:84]:
```js
console.log(favFoods.sort().reverse());
```

File: assignment/scripts/array-practice.js [hunk 3]
```diff
@@ -75,3 +75,14 @@ console.log('The animals are now', animalArray);
...
+
+// 4.g (STRETCH) TODO: Convert your array to a string
+//     putting the word "and" between each item.
+//     eg "tacos and pizza and pasta". Log the string.
```
Comment (eschwartz) [inline assignment/scripts/array-practice.js]:
```js
console.log(favFoods.join(' and '));
```

**Review by kdszafranski — CHANGES_REQUESTED** [review]
add in `.concat()` !

File: assignment/scripts/array-practice.js [hunk 4]
```diff
@@ -75,3 +75,20 @@ console.log('The animals are now', animalArray);
...
```
Comment (eschwartz) [inline assignment/scripts/array-practice.js:88]:
```js
console.log(favFoods.join(' and '));
```

File: assignment/scripts/array-practice.js [hunk 5]
```diff
@@ -75,3 +75,20 @@ console.log('The animals are now', animalArray);
...
+
+// 4.h (STRETCH) TODO: Make a new array that combines 
+//     the favorite foods array with the animals array.
+//     Then log the new array.
+//     It should look something like:
+//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
```
Comment (eschwartz) [inline assignment/scripts/array-practice.js:94]:
```js
console.log(favFoods.concat(animalsArray));
```

**Comment by eschwartz** [comment]
> add in .concat() !

https://github.com/PrimeAcademy/prime-pw-week-3-assignment-part-1/pull/2/files#diff-aff1dd259dec77cdfd14e04083011e0f06c597d4b87c8370114a3faea1ca33b9R90-R94

**Comment by kdszafranski** [comment]
Since these are stretch goals, let's remove the examples for some, maybe the last 2 and guide them with: research XYZ

**Review by kdszafranski — APPROVED** [review]
let's do it!

**Pull request merged**